src/comet/driver/light/photonic.py

```diff
@@ -0,0 +1,77 @@
+from comet.driver import Driver
+
+__all__ = ["Photonic"]
+
+
+class PhotonicDriver(Driver):
+    """Photonic LED light source base class"""
+
+    def __init__(self, resource):
+        self.resource = resource
+        self.resource.write_termination = "\r"
+        self.resource.read_termination = "\r"
+
+
+class Photonic(PhotonicDriver):
+    """Class for controlling Photonics F3000 LED light sources"""
+
+    @property
+    def brightness(self) -> int:
+        """Returns current brightness of light source
+            (0-100 percent)
+
+        Returns:
+            int . Brightness of light source in percent
+        """
+
+        self.resource.write("B?")
+        response = self.resource.read()
+
+        return int(response.replace("B", ""))
+
+    @brightness.setter
+    def brightness(self, brightness: int):
+        """Set brightness of light source in percent
+
+        Args:
+            brightness (int): Brightness to set in percent
+
+        Raises:
+            ValueError: Brightness outside of range [0,100]
+        """
+        brightness = int(brightness)
+        if brightness < 0 or brightness > 100:
+            raise ValueError("Brightness must be between 0 and 100")
```

Comment:
For convenience I suggest to simply mangle the value `brightness = max(0, min(100, brightness))` to reduce errors.

Reply:
See [cacbdc6](https://github.com/hephy-dd/comet/pull/75/commits/cacbdc6714a6ca0da40d1e291939c7c8bb92c110)